packages/sanity/src/desk/panes/loading/LoadingPane.tsx

```diff
@@ -18,8 +20,8 @@ interface LoadingPaneProps {
   tone?: CardTone
 }
 
-const DELAY = false
-const DEFAULT_MESSAGE = 'Loading…'
+const DELAY = true
+const DEFAULT_MESSAGE = 'panes.resolving.default-message'
```

Comment:
nit, should this be renamed to `DEFAULT_MESSAGE_KEY` ?

Reply:
> nit, should this be renamed to `DEFAULT_MESSAGE_KEY` ?

Good point!